apps/backend/app.py

```diff
@@ -43,11 +48,22 @@
 firebaseDb = firestore.client()
 firebaseBucket = storage.bucket("gladosbase.appspot.com")
 
+
+#MongoDB Objects
+mongoClient = pymongo.MongoClient('glados-mongodb', 27017, serverSelectionTimeoutMS=1000)
```

Comment:
Consider moving the port to a new env file listing, MONGO_PORT or similar

Reply:
do we have a process for updating the env vars? mongoDB port default is 27017, it might work without the argument explicitly defined

Reply:
https://github.com/AutomatingSciencePipeline/Monorepo/wiki/User-Guide#get-the-env-files

as long as you add it to the example version of the file and tell the rest of the dev team something got added, should be fine

Reply:
Or this, since it should only be accessed internally. @ehami says the frontend webserver should act as a middleman when people want to download stuff so they aren't directly asking the database for it.
https://www.mongodb.com/developer/languages/javascript/nextjs-with-mongodb/#querying-mongodb-with-next-js
```suggestion
DEFAULT_MONGO_PORT = 27017
mongoClient = pymongo.MongoClient('glados-mongodb', DEFAULT_MONGO_PORT, serverSelectionTimeoutMS=1000)
```